tests/integration/it_table_shapes_test.py

```diff
@@ -101,14 +102,9 @@ def test_edit(client):
     assert db_result.picture == update_data["picture"]
 
 
-@pytest.mark.skip(reason="create method doesn't have form.save()")
 def test_delete(client):
-    client.post(flask.url_for("table_shape.create"), data={
-        "description": "It's new shape",
-        "picture": "http://...."
-    })
-    identifier = models.TableShape.query.first().id
-
-    result = client.post(flask.url_for('table_shape.delete', id=identifier))
-    assert result.status_code == HTTPStatus.FOUND
-    assert models.TableShape.query.filter_by(id=identifier).count() == 0
+    table_shape = factories.TableShapeFactory()
```

Comment:
@vryazanov Shouldn't there be one method call here, like ``.getTableShape()``? As in, the Factory creates the table_shape, right?

Reply:
@amihaiemil actually factories work a bit different. You can have a look at example from library we are using: https://factoryboy.readthedocs.io/en/latest/#using-factories
I can change the code to `factories.TableShapeFactory.create()` to make it more understandable. wdyt?